myParser.py

```diff
@@ -67,6 +73,12 @@ def parse():
         type=int,
         help="Number of passengers (who are not seniors).")
 
+    parser.add_argument(
```

Comment:
Can you edit the Usage section of the README to give more information about this command?

Reply:
Will do